.jenkins/actions/run_tests_cmd.txt

```diff
@@ -0,0 +1,9 @@
+pip3 install --upgrade wheel
```

Comment:
Can this file have a `.sh` extension? It seems like a command file.

Reply:
I want the contents of this file to be sed into the sbatch script. If I call the script from the sbatch script, I think the output will likely be swallowed and we won't see it in jenkins.

Reply:
Ok, tried it out, and the output is not swallowed. switching to a nested script.